lib/client.js

```diff
@@ -8,6 +8,12 @@ var GiantSwarm = (function () {
 
   return {
 
+    _authorizedRequest: function() {
+      if (authToken) {
+        return request.set('Authorization', 'giantswarm ' + authToken);
+      }
+    },
```

Comment:
There should definitely be an `else`, but I'd rather use it to throw an exception. I'd like to add this after merge with PR #3, since there is a already a useful exception defined in that branch.